framework/platform/bpl/uci/scripts/platform_daemon.sh

```diff
@@ -54,7 +54,7 @@ static_init()
 
     # Kill DHCP server
     echo "Kill DHCP server"
-    ps -A | grep [d]ns | awk '{print $1;}' | xargs kill
+    pgrep [d]ns | xargs kill
```

Comment:
In fact, if `pgrep` is available, so will be `pkill` since they're both part of `procps`.
I see no reason to use `pgrep something | xargs kill` over `pkill something`.

Reply:
> In fact, if pgrep is available, so will be pkill since they're both part of procps.

Not necessarily. At the moment on the rax40 for example, Busybox has a variant of pgrep, but no pkill at all.

Reply:
This we can change - we decide what to have in the RAX40 busybox configuration, I added history for example. John has no problem that we add more capabilities to prplWrt.

Reply:
see my reply at:
https://github.com/prplfoundation/prplMesh/pull/1133#discussion_r409283040